test/gha/cloudformation/iam_cloudformation.yaml

```diff
@@ -18,7 +18,7 @@ Resources:
       ClientIdList:
         - sts.amazonaws.com
       ThumbprintList:
-        - 6938fd4d98bab03faadb97b34396831e3780aea1
+        - f879abce0008e4eb126e0097e46620f5aaae26ad
```

Comment:
Shouldn't this be both of them? https://github.blog/changelog/2023-06-27-github-actions-update-on-oidc-integration-with-aws/ and shouldn't the second one be `1c58a3a8518e8759bf075b76b750d4f2df264fcd`?

Reply:
Ah, good find. I didn't see that other blogpost

Reply:
This link only has the first two, where are you getting the last one from?

Reply:
Just removed it. Seems like there is another cert that was in that discussion thread but I'm not sure that it was neccesary.